actions/merkleroots/routes.go

```diff
@@ -0,0 +1,12 @@
+package merkleroots
+
+import (
+	"github.com/bitcoin-sv/spv-wallet/config"
+	"github.com/bitcoin-sv/spv-wallet/server/handlers"
+)
+
+// RegisterRoutes creates the specific package routes
+func RegisterRoutes(appConfig *config.AppConfig, handlersManager *handlers.Manager) {
+	group := handlersManager.Group(handlers.GroupAPI, "/merkleroots")
+	group.GET("", handlers.AsUserWithAppConfig(get, appConfig))
+}
```

Comment:
You don't have to add another `handlers` wrapper.
The user config is already provided and can be get using:
```
reqctx.AppConfig(c)
```

For example like this:
![image](https://github.com/user-attachments/assets/a8c5fa9a-e004-4f02-9363-651df7588098)

Reply:
Oh, allright I missed that. I will change accordingly and test